lib/src/layer/polygon_layer.dart

```diff
@@ -202,77 +271,12 @@ class PolygonPainter extends CustomPainter {
     canvas.drawCircle(offsets.last, radius, paint);
   }
 
-  void _paintLine(Canvas canvas, List<Offset> offsets, Paint paint) {
-    if (offsets.isEmpty) {
-      return;
-    }
-    final path = Path()..addPolygon(offsets, true);
-    canvas.drawPath(path, paint);
-  }
-
-  void _paintPolygon(Canvas canvas, Rect rect) {
-    final paint = Paint();
-
-    if (null != polygonOpt.holeOffsetsList) {
-      canvas.saveLayer(rect, paint);
-      paint.style = PaintingStyle.fill;
-
-      for (final offsets in polygonOpt.holeOffsetsList!) {
-        final path = Path();
-        path.addPolygon(offsets, true);
-        canvas.drawPath(path, paint);
-      }
-
-      paint
-        ..color = polygonOpt.color
-        ..blendMode = BlendMode.srcOut;
-
-      final path = Path();
-      path.addPolygon(polygonOpt.offsets, true);
-      canvas.drawPath(path, paint);
-
-      _paintBorder(canvas);
-
-      canvas.restore();
-    } else {
-      canvas.clipRect(rect);
-      paint
-        ..style =
-            polygonOpt.isFilled ? PaintingStyle.fill : PaintingStyle.stroke
-        ..color = polygonOpt.color;
-
-      final path = Path();
-      path.addPolygon(polygonOpt.offsets, true);
-      canvas.drawPath(path, paint);
-
-      _paintBorder(canvas);
-
-      if (polygonOpt.label != null) {
-        Label.paintText(
-          canvas,
-          polygonOpt.offsets,
-          polygonOpt.label,
-          polygonOpt.labelStyle,
-          rotationRad,
-          rotate: polygonOpt.rotateLabel,
-          labelPlacement: polygonOpt.labelPlacement,
-        );
-      }
-    }
-  }
-
   @override
   bool shouldRepaint(PolygonPainter oldDelegate) => false;
 
-  double _dist(Offset v, Offset w) {
-    return sqrt(_dist2(v, w));
-  }
+  double _dist(Offset v, Offset w) => sqrt(_dist2(v, w));
```

Comment:
I don't think we need these helpers? They are just overriding the standard library, most of which we have already imported in this file as well, which is less intuitive to devs.

Don't know about the standard lib but this excessive nesting will also just make longer call chains/slower code... if even just a tiny bit.

`_dist(v, w)` -> `(v - w).distance`
`_dist2(v, w)` -> `(v - w).distanceSquared`
`_sqr(x)` -> If need be we could just use `math.pow(x, 2)` or `x * x`

Reply:
Yeah these things are true, longer chains will cause more cache misses. Duplicating standard libraries is pointless here. I will fix that.